pkg/controller/add_cluster_conteroller.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
should this file be add_cluster_controller.go instead?

Reply:
lmao yes